static/src/stylesheets/module/external/_witness.scss

```diff
@@ -103,14 +103,31 @@
 
 .witness-logo {
     display: inline-block;
-    color: #ffffff;
-    font-weight: 700; 
     letter-spacing: -.04em;
     text-transform: lowercase;
+
+    .witness__button & {
+        color: #ffffff;
+        font-weight: 700;
+    }
+
+    .element-witness--source & {
+        color: #247eb3;
```

Comment:
We should change this `colour(news-support-2)` as it's basically the same and keeps it in the colour family.